python/hail/expr/expressions/typed_expressions.py

```diff
@@ -2500,9 +2521,8 @@ def global_position(self):
         :class:`.Expression` of type :py:data:`.tint64`
             Global base position of locus along the reference genome.
         """
-        reference_genome = self.dtype.reference_genome
-        return construct_expr(ApplyMethod('locusToGlobalPos({})'.format(reference_genome), self._ast),
-                              tint64, self._indices, self._aggregations)
+        name = 'locusToGlobalPos({})'.format(self.dtype.reference_genome)
```

Comment:
now that things are in the IR, it would be easy in the future to just add an IR `Str()` argument for the RG, right?

Reply:
I think we could do that, although we'd still need to lift out the reference genome arguments at compile time.